_posts/2017-06-09-notre-wiki.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Notre Wiki
+authors:
+  - laurent.bossavit
+---
+Dans certaines entreprises on distribue un « manuel de l'employé ». Il permet de s'orienter, de savoir quoi faire le premier jour et où poser ses affaires. Nous pensons qu'un « manuel de l'employé » est une excellente idée, et qu'elle est meilleure encore s'il est collaboratif et ouvert.
+
+À l'image d'un certain nombre d'entreprises du privé, comme [Basecamp](https://github.com/basecamp/handbook) ou [Valve](http://www.valvesoftware.com/company/Valve_Handbook_LowRes.pdf), nous avons dès le début décidé de le [rendre public](https://github.com/sgmap/beta.gouv.fr/wiki/). Si vous envisagez de travailler chez nous, ou de travailler avec nous, il vous permettra de savoir à quoi vous attendre, avant même le premier jour.
```

Comment:
Distinguons deux questions - faut-il référencer Valve et Basecamp, à quoi ma réponse est "oui, pour citation et social proof" - et quelle comparaison faire exactement; je souhaitais dire "comme eux le nôtre est rendu public", la correction apportée pour gommer l'implication que c'est une décision récente de notre part est venue brouiller le message.

> nous n'avons pas eu besoin qu'on nous apprend ça

Il me semble justement que si, qu'on l'a appris grâce à d'autres qui ont essuyé les plâtres, et qu'on a pu faire mieux (en le faisant tout de suite et par défaut) précisément parce qu'on a appris de cette façon. "Great ideas seem obvious with hindsight, but only in hindsight."